### PR TITLE
Remove TAM

### PR DESCRIPTION
**Pull request by krwq**
Removing old TestAssetsManager

### CODE REVIEW

**Comment by krwq** [comment]
@dotnet-bot test this please

**Comment by krwq** [comment]
@dotnet-bot test CentOS7.1 x64 Debug Build please

**Comment by krwq** [comment]
@dotnet-bot test Ubuntu x64 Release Build please

**Comment by krwq** [comment]
@jonsequitur @jgoshi @livarcocc @piotrpMSFT @piotroko

File: src/Microsoft.DotNet.TestFramework/TestAssetsManager.cs [hunk 1]
```diff
@@ -1,118 +0,0 @@
-﻿// Copyright (c) .NET Foundation and contributors. All rights reserved.
```
Comment (livarcocc) [inline src/Microsoft.DotNet.TestFramework/TestAssetsManager.cs]: Uhu!!

**Pull request merged**